ecf/cam/plots/jevs_href_grid2obs_ecnt_past31days_plots.ecf

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#PBS -N jevs_href_grid2obs_ecnt_plot
```

Comment:
It would certainly be helpful (and I think required) if the PBS -N line in your PBS card be the same thing that the ecf file is named.  For example, change this line to:

`#PBS -N jevs_href_gridtobs_ecnt_past31days_plots`

It would certainly make it a lot easier to identify the job by the name of the resulting .o file.  Otherwise a number of your plot jobs will have the same output filename, and thus it would be harder to identify the output file is a particular job crashed.  

Please check and be sure you do this for all your plot jobs.

Thanks!

Reply:
OK, I have changed them and updated in the github fork again